MantidQt/CustomInterfaces/src/Indirect/ResNorm.cpp

```diff
@@ -173,8 +173,11 @@ void ResNorm::handleAlgorithmComplete(bool error) {
   if (plotOptions == "Fit" || plotOptions == "All")
     plotSpectrum(fitWsName, 0, 1);
 
-  loadFile(m_uiForm.dsResolution->getFullFilePath(),
-           m_uiForm.dsResolution->getCurrentDataName());
+  if (!AnalysisDataService::Instance().doesExist(
+          m_uiForm.dsResolution->getCurrentDataName().toStdString())) {
```

Comment:
you can use `resWsName.toStdString()` here. Also, do we know why we are only checking if the Res workspace is present in the ADS? and if yes, should we be checking if the Vanadium workspace is also available?